python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/router-bgp.j2

```diff
@@ -1031,10 +1031,15 @@ ASN Notation: {{ router_bgp.as_notation | arista.avd.default('asplain') }}
 {%             set multicast_out = [] %}
 {%             do multicast_out.append("IPv4: " ~ multicast) %}
 {%             do multicast_out.append("Transit: " ~ multicast_transit) %}
+{%             set graceful_restart_time = vrf.graceful_restart.restart_time | arista.avd.default("-") %}
+{%             set graceful_restart_stale = vrf.graceful_restart.stalepath_time | arista.avd.default("-") %}
+{%             set graceful_out = [] %}
+{%             do graceful_out.append("Restart-Time: " ~ graceful_restart_time) %}
+{%             do graceful_out.append("Stalepath-Time: " ~ graceful_restart_stale) %}
```

Comment:
IMO the output was too cluttered. Especially for most cases where nothing was set.

```suggestion
{%             set graceful_restart = vrf.graceful_restart.enabled | arista.avd.default("-") %}
{%             if vrf.graceful_restart.enabled | arista.avd.default(false) and vrf.graceful_restart.restart_time is arista.avd.defined %}
{%                 set graceful_restart = graceful_restart ~ " ({{ vrf.graceful_restart.restart_time }}s)" &}
{%             endif %}
```

Reply:
ah, now i got you. you dont want to the stalepath option in the table. 

let me update that